python/catkin/builder.py

```diff
@@ -702,12 +704,13 @@ def build_package(
 
 
 def get_new_env(package, develspace, installspace, install, last_env, destdir=None):
+    env_script = 'env.bat' if platform.system() is 'Windows' else 'env.sh'
```

Comment:
Should the condition use `sys.platform == 'win32'` instead to not use `.bat` when using `cygwin`?

Reply:
change to use sys.platform